keel-clouddriver/src/main/kotlin/com/netflix/spinnaker/keel/clouddriver/model/ActiveServerGroup.kt

```diff
@@ -173,6 +210,25 @@ data class LaunchConfig(
   val instanceMonitoring: InstanceMonitoring
 )
 
+data class LaunchTemplate(
```

Comment:
question: can `LaunchTemplateData` be at the top level, meaning just move all of its fields to `LaunchTemplate`?

Reply:
The challenge is the shape of the clouddriver response is:

```json
{
 "launchTemplate": {
   "launchTemplateData": {

   }
  }
}
```

While we probably could write some Jackson-related code so that these classes don't have to exactly match the shape of the response, my inclination is to keep the Jackson mapping logic as simple as possible.

Reply:
oh I see, makes sense!